apps/docs/src/app/core/component-docs/fixed-card-layout/examples/mobile/fixed-card-layout-mobile-examples.component.ts

```diff
@@ -0,0 +1,32 @@
+import { ChangeDetectorRef, Component, ViewChild } from '@angular/core';
+import { FixedCardLayoutComponent } from '@fundamental-ngx/core';
+
+const portraitWidth = 385; // width in pixel for phone
+const landscapeWidth = 667;
+
+@Component({
+    selector: 'fd-fixed-card-layout-mobile-examples',
+    templateUrl: './fixed-card-layout-mobile-examples.component.html'
+})
+export class FixedCardLayoutMobileExampleComponent {
+    /** @hidden */
+    @ViewChild(FixedCardLayoutComponent)
+    layout: FixedCardLayoutComponent;
+
+    mobileWidth: number = portraitWidth;
+    mobileHeight: number = landscapeWidth;
+
+    constructor(private readonly _cd: ChangeDetectorRef) {}
+
+    public changeOrientation(orientation: string): void {
+        if (orientation === 'portrait') {
+            this.mobileWidth = portraitWidth;
+            this.mobileHeight = landscapeWidth;
+        } else {
+            this.mobileWidth = landscapeWidth;
+            this.mobileHeight = portraitWidth;
+        }
+        this._cd.detectChanges();
```

Comment:
That wont work in case of `ChangeDetectionStrategy.OnPush` here.
we have to add `markForCheck` in `onResize` method to make it reliable for invoiking outside.

Reply:
Btw let's use `OnPush` strategy even for example components

Reply:
fixed